client/src/views/Settings/Tabs/EditPrivacy/index.vue

```diff
@@ -75,21 +75,197 @@
         </q-item-section>
       </q-item>
 
+      <q-separator spaced />
+      <q-item-label header>Black list</q-item-label>
+      <div class="row justify-between">
+        <div
+          id="input-search"
+          ref="search"
+          class="col"
+          >
+          <q-input v-model="nickname" @input="searchByNick" />
+          <block-search
+            v-if="showSearch"
+            id="search-result"
+            :results="nicknameSearchResults"
+            @handleChoose="handleChoose"
+          />
+        </div>
+        <q-btn
+          @click="blockUser"
+          id="block-button"
+          class="col"
+          :disabled='!enableBlock'
+        >Block</q-btn>
+      </div>
+      <p v-if="userToBlockNick">{{userToBlockNick}}  <q-btn
+        round
+        color = 'teal'
+        size = 'xs'
+        icon = 'close'
+        @click = "userToBlockNick = ''"
+      /></p>
+      <p id="error-message" v-if="errorMessage">{{errorMessage}}  <q-btn
+        round
+        color = 'red'
+        size = 'xs'
+        icon = 'close'
+        @click = "errorMessage = ''"
+      /></p>
+      <blocked-list id="blocked-list" @unblock="unblockUser" :list="blockedProfilesList"></blocked-list>
     </q-list>
   </div>
 </template>
 
 <script>
+import BlockSearch from '@/components/BlockSearch.vue'
+import BlockedList from '@/components/BlockedList.vue'
+import debounce from 'lodash/debounce'
```

Comment:
For libraries please follow this type of import whenever possible:
_import {debounce} from 'lodash'_
it will be much easier to add new imports when needed